src/OrbitQt/HistogramWidget.cpp

```diff
@@ -207,11 +203,7 @@ void HistogramWidget::paintEvent(QPaintEvent* /*event*/) {
   DrawHorizontalAxis(painter, axes_intersection, histogram, horizontal_axis_length, MinValue());
   DrawVerticalAxis(painter, axes_intersection, vertical_axis_length, max_freq);
 
-  if (IsSelectionActive()) {
-    DrawTitle(painter, absl::StrFormat("Selection over %u points", histogram.data_set_size));
-  } else {
-    DrawTitle(painter, function_data_->name);
-  }
+  emit SignalTitleChange(GetTitle());
```

Comment:
This is also in paintEvent and needs to be moved. Sorry, I forgot to make a separate comment.